src/raderon.php

```diff
@@ -173,7 +170,12 @@
 
 //Calculate some variables that will be used
 $forcadoplayer = ceil(($player->strength + $player->atkbonus['effectiveness'] + ($player->atkbonus['item_bonus'] * 2) + $pbonusfor) * $multipleatk);
-$agilidadedoplayer = ceil($player->agility + $player->agibonus6['effectiveness'] + ($player->agibonus6['item_bonus'] * 2) + $pbonusagi);
+$agilidadedoplayer = ceil($player->agility +
```

Comment:
Here you correctly added the bonus stats of the item. No adjustments are needed; I just commented so you can see what was adjusted in the others.